src/main/java/com/alkemy/ong/controller/ActivityController.java

```diff
@@ -1,4 +1,27 @@
 package com.alkemy.ong.controller;
 
+import com.alkemy.ong.dto.ActivityDto;
+import com.alkemy.ong.service.ActivityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/activities")
 public class ActivityController {
+
+    private final ActivityService activityService;
+
+    @PostMapping("/create")
+    public ResponseEntity<ActivityDto> addActivity(@Validated @RequestBody ActivityDto activityDto) {
+        ActivityDto savedActivity = activityService.addActivity(activityDto);
```

Comment:
Suggestion try return new ResponseEntity<>(HttpStatus.UNPROCESSABLE_ENTITY); (for the empty and null exceptions)
you can see it implemented in Walter PR of News entity creation method

Reply:
as it is a method to create an entity, the validations are in the Dto, with which in case of any empty or null parameter, this class will be in charge of processing the error